core-web/libs/dot-primeng-theme-styles/src/scss/_variables.scss

```diff
@@ -219,3 +220,6 @@ $tabView-scroll-btn-width: 40px;
 
 // Tag
 $tag-radius-md: 7px;
+
+// Experiments
+$container-reports-max: 1100px;
```

Comment:
Move this to it own context, this is not a global var. I am fixing some of this in the unify SCSS vars PR

Reply:
moved